### PR TITLE
Fix issue 125

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue #125 .

### Summary of Changes
This PR does two things:
1. Only show IP column in the email if IP Logging is enabled
2. Use Text to show IP column to avoid a language item displayed as reported in #125

### Testing Instructions
1. See https://github.com/joomla-projects/privacy-framework/issues/125 for the issue description
2. Apply patch, confirm that the issue is fixed
3. Check and confirm when IP logging is disabled, the IP column is hided in email

### CODE REVIEW

File: administrator/components/com_actionlogs/layouts/logstable.php [hunk 1]
```diff
@@ -25,7 +26,9 @@
 		<th><?php echo JText::_('COM_ACTIONLOGS_DATE'); ?></th>
 		<th><?php echo JText::_('COM_ACTIONLOGS_EXTENSION'); ?></th>
 		<th><?php echo JText::_('COM_ACTIONLOGS_NAME'); ?></th>
-		<th><?php echo JText::_('COM_ACTIONLOGS_IP_ADDRESS'); ?></th>
+		<?php if ($showIpColumn) : ?>
+            <th><?php echo JText::_('COM_ACTIONLOGS_IP_ADDRESS'); ?></th>
```
Comment (mbabker) [inline administrator/components/com_actionlogs/layouts/logstable.php]: Change spaces to tabs.

File: administrator/components/com_actionlogs/layouts/logstable.php [hunk 2]
```diff
@@ -25,22 +26,26 @@
...
-                    <td><?php echo $message->message; ?></td>
-                    <td><?php echo $message->log_date; ?></td>
-                    <td><?php echo $message->extension; ?></td>
-                    <td><?php echo $displayData['username']; ?></td>
-                    <td><?php echo $message->ip_address; ?></td>
-                </tr>
-            <?php
-            }
-        ?>
+		<?php
```
Comment (Quy) [inline administrator/components/com_actionlogs/layouts/logstable.php]: Please change to `<?php foreach ($messages as $message) : ?>`

File: administrator/components/com_actionlogs/layouts/logstable.php [hunk 3]
```diff
@@ -25,22 +26,26 @@
...
+				<tr>
+					<td><?php echo $message->message; ?></td>
+					<td><?php echo $message->log_date; ?></td>
+					<td><?php echo $message->extension; ?></td>
+					<td><?php echo $displayData['username']; ?></td>
+					<?php if ($showIpColumn) : ?>
+						<td><?php echo JText::_($message->ip_address); ?></td>
+					<?php endif; ?>
+				</tr>
+			<?php
```
Comment (Quy) [inline administrator/components/com_actionlogs/layouts/logstable.php]: Change to `<?php endforeach; ?>`

Comment (joomdonation) [reply administrator/components/com_actionlogs/layouts/logstable.php]: Thanks, changed it (not sure if it's OK now, I am not familiar with this code style)

**Pull request merged**